src/components/results/recommendations.vue

```diff
@@ -1,25 +1,24 @@
 <template>
   <div
-    v-if="$x.recommendations.length > 0"
+    v-if="!$x.query.searchBox && $x.recommendations.length > 0"
```

Comment:
Adding this check here, we are not showing the top clicked on no results situations.